pkg/cmd/container/list.go

```diff
@@ -107,6 +107,7 @@ func (x *ListItem) Label(s string) string {
 
 func prepareContainers(ctx context.Context, client *containerd.Client, containers []containerd.Container, options types.ContainerListOptions) ([]ListItem, error) {
 	listItems := make([]ListItem, len(containers))
+	snapshottersCache := map[string]containerdutil.Snapshotter{}
```

Comment:
Here, we are potentially dealing with multiple snapshotters, so, keeping them around for the duration of the function.

By the way, we are currently not dealing with mutiple different snapshotters for images. Should we?